examples/demo-async-dns-cluster.php

```diff
@@ -17,13 +17,20 @@
 
 require __DIR__ . '/../vendor/autoload.php';
 
+/**
+ * Start docker/local-3-node-dns-cluster and run this script from host machine.
+ * This is because the cluster advertises as 127.0.0.1, which does not resolve
+ * to the event store in the PHP container, if you run it in Docker.
+ */
+
 Loop::run(function () {
-    $builder = new ConnectionSettingsBuilder();
-    $builder->setClusterDns('escluster.net');
-    $builder->setClusterGossipPort(2113);
+    $settings = ConnectionSettings::create()
+        ->setClusterDns('127.0.0.1')
```

Comment:
I added `escluster.net` to my `hosts` file to test this. It's not a great way and requires an additional manual step, but at least tests the cluster dns stuff.